uber/site_sections/preregistration.py

```diff
@@ -387,8 +387,14 @@ def prereg_payment(self, session, payment_id=None, stripeToken=None):
             if attendee.badges:
                 pc_group = session.create_promo_code_group(attendee, attendee.name, int(attendee.badges) - 1)
                 session.add(pc_group)
+                session.add(session.create_receipt_item(
```

Comment:
Reviewing it, this may be unnecessary. I originally had `session.add_receipt_items_by_model(charge, attendee)` run before I committed the promo code group to the session, so it wasn't catching the attendee's promo code group (because they didn't have one yet). Something to look into later.